deployer/src/deployer/update_lambda_functions.py

```diff
@@ -71,7 +71,7 @@ def update(lambda_function_dir, dry_run=False):
     function_hash = function_info["Configuration"]["CodeSha256"]
     zip_file_bytes = zip_filename.read_bytes()
     # If nothing has changed, don't do anything.
-    if get_sha256_hash(zip_file_bytes) != function_hash:
+    if force or get_sha256_hash(zip_file_bytes) != function_hash:
```

Comment:
Why should it not work? I added the force flag *because* it was working. 😉

Reply:
Oh, you're right, timestamps are not the same in the workflow run, hence the many versions. Locally, it works because the timestamps only change when the files actually change.

See also: https://tanzu.vmware.com/content/blog/barriers-to-deterministic-reproducible-zip-files

Reply:
@fiji-flo ba787c688 should fix this.